exonum/src/node/mod.rs

```diff
@@ -142,6 +144,8 @@ pub struct NodeApiConfig {
     pub public_api_address: Option<SocketAddr>,
     /// Listen address for private api endpoints.
     pub private_api_address: Option<SocketAddr>,
+    /// Set CORS part of response header for browser-based clients.
```

Comment:
Nitpick: is "set" a verb here? It looks out of place, imo. Also, not sure why "browser-based" is explicitly mentioned here. Isn't it the case that CORS headers would be returned for *all* HTTP requests? I'd suggest to use something like

```rust
/// Cross-origin resource sharing ([CORS][cors]) options for responses returned
/// by API handlers.
///
/// [cors]: https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS
```